pepper-apis/dsm-core/src/main/java/org/broadinstitute/lddp/util/DeliveryAddress.java

```diff
@@ -49,30 +55,7 @@ public DeliveryAddress(String street1, String street2, String city, String state
         this.phone = phone;
     }
 
-    private DeliveryAddress(@NonNull Address address) {
-        this.id = address.getId();
-        this.name = address.getName();
-        this.street1 = address.getStreet1();
-        this.street2 = address.getStreet2();
-        this.city = address.getCity();
-        this.state = address.getState();
-        this.zip = address.getZip();
-        this.country = address.getCountry();
-        this.phone = address.getPhone();
-    }
-
-    public static DeliveryAddress populateFromEasyPost(@NonNull String id) {
-        DeliveryAddress deliveryAddress = null;
-        try {
-            Address address = Address.retrieve(id);
-            deliveryAddress = new DeliveryAddress(address);
-        } catch (Exception ex) {
-            logger.error(LOG_PREFIX + "An error occurred during address retrieval.", ex);
-        }
-        return deliveryAddress;
-    }
-
-    public void validate() {
+    public boolean validate() {
```

Comment:
Returned the `valid` value for convenience.

Reply:
If you now understand what this method/class does it would be great to add a comment. IOW, what does the EasyPost validation do? Are we assured of having a valid shipping address, or is this just checking that the proper fields are included?